money-api/src/main/java/com/comcast/money/api/SpanFactory.java

```diff
@@ -20,5 +20,9 @@
 
     Span newSpan(String spanName);
 
-    Span newSpan(String spanName, boolean sticky);
+    Span newSpan(SpanId spanId, String spanName);
+
+    Span childSpan(String childName, Span span);
```

Comment:
So if we want to make a child, we send in the parent to the factory? Seems like there might be an easier way... like the parent keeps a reference to the factory that created it (or something). Not a huge deal, just feels a little awkward from an API perspective.